### ISSUE TITLE
FR: Hide user account 

### ISSUE THREAD

**Issue by redshirtdave** [gregneagle/pycreateuserpkg#17]
Incorporate using options to hide account in postinstall, as per https://support.apple.com/en-au/HT203998

**Comment by gregneagle** [comment]
This is not something I need, so I'm unlikely to work on it any time soon.

**Comment by redshirtdave** [comment]
np, cheers!

**Comment by gregneagle** [comment]
Adding the isHidden attribute seems very straightforward, and I'd accept a PR that did that. You can already specify a hidden home directory location. I don't think accounts created via pycreateuserpkg packages get an entry in the list of SharePoints.

**Comment by redshirtdave** [comment]
I've tested setting the IsHidden flag to 1 after creating an account with your script. It has the added benefit of hiding the account from loginwindow and 'Users & Groups' panel in System Preferences. Hidden home directory doesn't achieve this alone. So this would be an improvement for those wanting to hide the account. ps. I don't think a UID below 500 is best practice anymore.

I think this would be nice to include, otherwise it's easy to activate post install with any management system.

I initially asked to consider removing SharePoints, but removed the request once I have confirmed it isn't populated.
